apps/userdashboard/views.py

```diff
@@ -40,6 +40,13 @@ def actions(self):
             actor=self.request.user,
         ).exclude_updates()
 
+    @property
+    def projects_carousel(self):
+        sorted_active_projects, sorted_future_projects, sorted_past_projects =\
+            self.request.user.get_projects_follow_list()
```

Comment:
I guess, we do filter out the private projects here? In apps.users.models? But we need them here. Can we just filter them out for the public profile then @Rineee ?

Reply:
Yes, thats true, we filter out the private projects in user.get_projects_follow_list(). I will add a parameter to it, so we can only filter them out if we want to.. but lets merge it here like this and I'll add a task?